opentracing/standard/raw.py

```diff
@@ -30,13 +30,13 @@ def __init__(self, trace_context, operation_name):
         self.operation_name = operation_name
         self.start_time = None
         self.duration = None
-        self.tags = None  # array of (key,vale) pairs when initialized
+        self.tags = None  # dict when initialized
         self.logs = None  # array of RawLog's when initialized
 
-    def add_tag(self, key, value):
+    def set_tag(self, key, value):
         if self.tags is None:
-            self.tags = []
-        self.tags.append((key, value))
```

Comment:
Yeah. I'm not volunteering to write this right now (as much as I'd enjoy it!), but we could have a super-simple polyglot standalone "OpenTracing" tracer service, too... basically just a reference impl that stores things in memory and offers few features beyond chronological indexing and a simple trace viewer. But the "standard" tracer is sounding less useful than some handy util code for each platform.